ci/si_install_deps.sh

```diff
@@ -27,7 +27,18 @@ if ! command -v timeout >/dev/null 2>&1; then
     if [ "$PLATFORM" == 'Darwin' ]; then
         brew install coreutils
         PATH="/usr/local/opt/coreutils/libexec/gnubin:$PATH"
+        export PATH
+    fi
+fi
+
+# Ensure the latest DC/OS CLI is available
+if ! command -v dcos >/dev/null 2>&1; then
+    if [ "$PLATFORM" == 'Darwin' ]; then
+	curl -o /usr/local/bin/dcos https://downloads.dcos.io/binaries/cli/darwin/x86-64/dcos-1.11/dcos
```

Comment:
We still use the CLI but the latest version and it does *not* come with Shakedown since it should become a lib.